### PR TITLE
spu: df[x] instructions

### PR DESCRIPTION
**Pull request by elad335**

### CODE REVIEW

File: rpcs3/Emu/Cell/SPUInterpreter.cpp [hunk 1]
```diff
@@ -757,7 +757,7 @@ void spu_interpreter_fast::FCGT(SPUThread& spu, spu_opcode_t op)
 
 void spu_interpreter::DFCGT(SPUThread& spu, spu_opcode_t op)
 {
-	fmt::throw_exception("Unexpected instruction" HERE);
+	spu.gpr[op.rt].vd = _mm_cmpgt_pd(ra, rb);
```
Comment (isJuhn) [inline rpcs3/Emu/Cell/SPUInterpreter.cpp]: Shouldn't it be op.ra and op.rb?

File: rpcs3/Emu/Cell/SPUInterpreter.cpp [hunk 2]
```diff
@@ -976,7 +976,7 @@ void spu_interpreter_fast::FCEQ(SPUThread& spu, spu_opcode_t op)
 
 void spu_interpreter::DFCEQ(SPUThread& spu, spu_opcode_t op)
 {
-	fmt::throw_exception("Unexpected instruction" HERE);
+  	spu.gpr[op.rt].vd = _mm_cmpeq_pd(ra, rb);
```
Comment (isJuhn) [inline rpcs3/Emu/Cell/SPUInterpreter.cpp]: same thing here?

**Comment by Nekotekina** [comment]
SPU 1.2 shouldn't be implemented in PS3.
May need hardware test but I really doubt it.

**Comment by flash-fire** [comment]
Do you need help setting up a local appveyor? It's super easy!  If VS builds, appveyor should build too. Why submit code as a PR? It makes you look like an idiot cause everyone knows you couldn't even test this stuff lol

**Comment by elad335** [comment]
@flash-fire thanks but I aleady set that up @Nekotekina so I'll open another pull for the other things I wanted to do and leave this one open until you decide otherwise.

**Comment by clienthax** [comment]
@elad335 Do you have any HW tests for this? i can run them if needed

**Comment by elad335** [comment]
@clienthax I'll make a dftsv test and send you , other commits don't need testing.

**Comment by elad335** [comment]
closed as being irrelevant for the ps3

**Pull request closed without merging**